packages/web/src/js/Collapse.ts

```diff
@@ -59,109 +61,149 @@ class Collapse extends BaseComponent {
     return `${this.NAME}`;
   }
 
-  onResize() {
-    this.state.width = window.innerWidth;
-    this.updateTriggerElementHandler();
-    this.updateCollapsibleElementHandler();
+  hideParentSiblings(trigger: HTMLElement) {
+    if (this.meta.parent) {
+      const parentElement = SelectorEngine.findOne(this.meta.parent);
+      const children = parentElement?.children;
+
+      if (!children) {
+        return;
+      }
+
+      for (const item of children) {
+        const itemTrigger = SelectorEngine.findOne(`[${NAME_DATA_TOGGLE}="${NAME}"]`, item);
+        const instance = Collapse.getInstance(itemTrigger);
+
+        if (instance?.state?.open && itemTrigger !== trigger) {
+          instance?.hide();
+        }
+      }
+    }
   }
 
-  appendNodeToParentHandler() {
-    if (this.state.collapsed) {
+  async appendNodeToParent() {
```

Comment:
There is no existing example for this functionality. Please, create an issue for this or add an example into HTML. For now, this code is unreachable in examples.

Reply:
@literat When you take a look on Collapse example and title for 'Collapse with hide trigger when expanded'

Reply:
We've discussed this on call. I've found it, thanks.